### PR TITLE
media-sound/drumkv1: Fix low res icon, update copyright header

### PR DESCRIPTION
**Pull request by simonvanderveldt**
Small change, adds the same fix as I already did in #30 and as it has been fixed in upstream `master` to the 0.8.2 version of drumkv

### CODE REVIEW

**Comment by mrVanDalo** [comment]
It's hard to see what is new now. Did you just copy the file ?

**Comment by simonvanderveldt** [comment]
@mrVanDalo Copied the file and added the `src_prepare`

File: media-sound/drumkv1/drumkv1-0.8.2.ebuild [hunk 1]
```diff
@@ -1 +0,0 @@
-drumkv1-9999.ebuild
```
Comment (marcan) [inline media-sound/drumkv1/drumkv1-0.8.2.ebuild]: While you're here, mind fixing the copyright year? Seems we missed it last time.

Comment (simonvanderveldt) [inline media-sound/drumkv1/drumkv1-0.8.2.ebuild]: Sure, will do!

**Pull request merged**